page/news.php

```diff
@@ -65,25 +64,46 @@
 
             <div class="pb-5">
                 <h5><dt><b>POPULAR POST</b></dt></h5>
-                <div class="row popular-post my-4">
-                    <div class="col-md-3">
-                        <a href=""><img src="img/berita/1._Penandatanganan_Kontrak_Tank_Boat_2_.jpg" class="img-fluid img-berita" alt=""></a>
-                    </div>
-                    <div class="col-md-9">
-                        <div class="text-warning font-weight-bold tanggal">Tanggal</div>
-                        <a href="" class="font-weight-bold judul">Pindad Resmikan Lini Baru Fasilitas Produksi Sistem Senjata</a>
-                    </div>
-                </div>
-                <div class="row popular-post my-4">
-                    <div class="col-md-3">
-                        <a href=""><img src="img/berita/1._Penandatanganan_Kontrak_Tank_Boat_2_.jpg" class="img-fluid img-berita" alt=""></a>
-                    </div>
-                    <div class="col-md-9">
-                        <div class="text-warning font-weight-bold tanggal">Tanggal</div>
-                        <a href="" class="font-weight-bold judul">Pindad Resmikan Lini Baru Fasilitas Produksi Sistem Senjata</a>
+
+                <?php
+
+                $popular_post = mysqli_query($connection, "select * from berita order by tanggal desc");
+                $i = 0;
+                while($data = mysqli_fetch_array($popular_post)){
+                    if($i == 4){
+                        break;
+                    }
+                    if($data[4]==null || $data[4]==""){
+                        $thumbnail = "#";
+                    }else{
+                        $thumbnail = "news/".explode("-", $data[4])[0]."/gambar/".explode("-", $data[4])[1];
+                    }
+                    ?>
+
+                    <div class="row popular-post my-4">
+                        <div class="col-md-3">
+                            <a href="index.php?page=view-news&id=<?php echo $data[0] ?>"><img src="<?php echo $thumbnail ?>" alt=""></a>
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)